test/Directory.Build.props

```diff
@@ -8,7 +8,7 @@
     <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net461'">win-x64</RuntimeIdentifier>
     <IsPackable>false</IsPackable>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
```

Comment:
Why is this necessary?

Reply:
It was needed to create the app.config files in the publish folders. I have since switched to not need to actually run the `publish` command so I will remove it.